docs/source/markdown/podman-network-create.1.md

```diff
@@ -39,6 +39,9 @@ Define a gateway for the subnet. If you want to provide a gateway address, you m
 *subnet* option. Can be specified multiple times.
 The argument order of the **--subnet**, **--gateway** and **--ip-range** options must match.
 
+#### **--ignore**
+Ignore the create request if a network with the same name already exists instead of failing
```

Comment:
Probably worth mentioning that only the name is checked, so `podman network create --driver macvlan testnet` will still exit cleanly if `testnet` exists but isn't macvlan, or similar.

Reply:
Please see the line I've added

Reply:
Looks fine